org.musicbrainz.Picard.json

```diff
@@ -1,54 +1,61 @@
 {
     "id": "org.musicbrainz.Picard",
-    "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
-    "sdk": "org.freedesktop.Sdk",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
     "command": "picard",
     "rename-desktop-file": "picard.desktop",
     "rename-icon": "picard",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
         "--share=network",
         "--filesystem=home",
-        "--filesystem=xdg-music",
-        "--talk-name=org.gnome.GConf"
+        "--filesystem=xdg-music"
     ],
     "modules": [
-        "shared-modules/qt4/qt4-4.8.7-minimal.json",
         {
-            "name": "sip",
+            "name": "pyqt",
+            "config-opts": [
+                "--disable-static",
+                "--enable-x11"
+            ],
```

Comment:
I recently removed these config-opts from the Picard 1 manifest, they have never been useful (the configure script ignores them entirely).

I guess you brought them back when you rebased. Can you remove them please?